pkg/gcc/arrival_group_accumulator.go

```diff
@@ -49,6 +49,9 @@ func (a *arrivalGroupAccumulator) run(in <-chan []cc.Acknowledgment, agWriter fu
 					continue
 				}
 
+				// fmt.Println("different group, dep delay:", interDepartureTimePkt(group, next),
+				// 	", interArrivalTimePkt:", interArrivalTimePkt(group, next),
+				// 	",interGroupDelayVariationPkt:", interGroupDelayVariationPkt(group, next))
```

Comment:
One more thing I noticed, c++ has one more check for detecting bursts, they don't consider packets more than 100ms behind - `arrival_time - current_timestamp_group_.first_arrival < kMaxBurstDuration)` 
This is additional check along with `interArrivalTimePkt` and `interGroupDelayVariationPkt`

Reply:
Burst duration greater than 100ms is extremely rare, I saw that in C++, I did not add it. We can make a change for this later. 
C++ code is handling lot more border conditions
For instance, they couple of checks for situations when the subscriber clock has a jump and starts giving bad arrival times. 
Wanted to keep these changes to the minimum